src/ixdat/readers/biologic.py

```diff
@@ -34,6 +34,32 @@
 }
 
 
+def fix_WE_potential(measurement):
+    """Fix column of zeros in "<Ewe>/V" sometimes exported by EC Lab for CP measurements.
+
+    Some Biologic potentiostats / EC-Lab versions sometimes export a column of zeros for
+    "<Ewe>/V" in the .mpt files in chronopotentiometry measurements. This function
+    replaces the series of zeros with the correct potential by adding the counter
+    electrode potential ("<Ece>/V") and cell potential ("Ewe-Ece/V").
+
+    Args:
+        measurement(ECMeasurement): The measurement with the column to be replaced
+    """
+    WE_series = measurement["<Ewe>/V"]
+    CE_data = measurement.grab_for_t("<Ece>/V", WE_series.t)
```

Comment:
This is only run manually. I've specified that now in the doc string, and added an error message in case it is used on a measurement for which the counter electrode potential was not recorded.